### PR TITLE
main file

### PR DESCRIPTION
**Pull request by SamvruthaShaji**
The main file of brick breaker game
JFrame package is used and created a object for gameplay class

### CODE REVIEW

File: Main.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package brickBraker;
```
Comment (anandbaburajan) [inline Main.java]: Maybe you meant `brickBreaker`?

File: Main.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import javax.swing.JFrame;
+
+public class Main {
+
+	public static void main(String[] args) {
+    Gameplay gamePlay = new Gameplay();		
+    JFrame obj = new JFrame();
+    obj.setBounds(10,10,700,600);
+	obj.setTitle("Breakout Ball");
```
Comment (anandbaburajan) [inline Main.java]: Can the alignment be fixed here? Maybe you can use a Java formatter/indenter.

**Review by anandbaburajan — CHANGES_REQUESTED** [review]
@SamvruthaShaji Thanks! I've added two comments after which the PR can be merged!

File: Main.java (hunk 1 above)
Comment (SamvruthaShaji) [reply Main.java]: Yes, It's brickBreaker 
Want to change this?

Comment (anandbaburajan) [reply Main.java]: You can go ahead and push another commit with the correct spelling.

File: Main.java (hunk 2 above)
Comment (SamvruthaShaji) [reply Main.java]: Yes,I will rectify this two issues and push another one

File: Main.java (hunk 1 above)
Comment (SamvruthaShaji) [reply Main.java]: Okay

**Comment by anandbaburajan** [comment]
Looks good, thanks!

**Pull request merged**